bin/growpart

```diff
@@ -162,7 +162,7 @@ lock_disk() {
 	# Do not use --nonblock or --timeout as udev may be already processing
 	# the disk and we must wait until it has released the disk to
 	# proceed.  Failure to obtain exclusive lock is fatal to growpart.
-	rq flock flock --exclusive $FLOCK_DISK_FD ||
+	rq flock flock -x $FLOCK_DISK_FD ||
```

Comment:
would we entertain putting the value in a variable
FLOCK_EXCLUSIVE="-x"  so the readers don't need to look up the short name of the flag?

Reply:
I guess that'd have to be upstream, this is just the changes for the new snapshot.

Reply:
Maybe a comment could be enough, I can make a PR "upstream" but I'd keep this as it is for the snapshot.